src/gerbil/runtime/mop.ss

```diff
@@ -22,17 +23,27 @@ namespace: #f
 ;;
 ;; Gerbil rtd on gambit
 ;; ##structure ##type-type
-;;  1  ##type-id
-;;  2  ##type-name
-;;  3  ##type-flags
-;;  4  ##type-super
-;;  5  ##type-fields
-;;  6                       type-descriptor-mixin
-;;  7                       type-descriptor-fields
-;;  8                       type-descriptor-plist
-;;  9                       type-descriptor-ctor
-;; 10                       type-descriptor-slots
-;; 11                       type-descriptor-methods
+;;  1  ##type-id                                  :: Symbol (sometimes uninterned)
+;;  2  ##type-name                                :: Symbol
+;;  3  ##type-flags                               :: Fixnum
+;;  4  ##type-super                               :: (OrFalse StructTypeDescriptor)
+;;  5  ##type-fields                              :: (Vector [Symbol ??? ???] ...)
+;;  6                 type-descriptor-mixin       :: (List TypeDescriptor)
+;;  7                 type-descriptor-fields      :: Fixnum
+;;  8                 type-descriptor-plist       :: AList (!)
+;;  9                 type-descriptor-ctor        :: Symbol
```

Comment:
Maybe Symbol

Reply:
I use `Maybe` when the null case is `#!void` and `OrFalse` when the null case is `#f`. I assume it's `OrFalse` here.